{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Error: SequelizeConnectionError\n    at Object.fetchUserByEmail (file:///home/emmanuel/kitchen/work-project/questn/src/modules/auth/repository.js:30:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async ensureUniqueUser (file:///home/emmanuel/kitchen/work-project/questn/src/middlewares/auth.js:45:26)"}
{"level":"error","message":"Database connection failed: SequelizeConnectionError\n    at Client._connectionCallback (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/sequelize@6.37.3_pg@8.12.0/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (node:events:518:28)\n    at Socket.reportStreamError (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/pg@8.12.0/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (node:events:518:28)\n    at emitErrorNT (node:internal/streams/destroy:169:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:128:3)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)"}
{"level":"error","message":"Error connecting to database mongoose is not defined","stack":"ReferenceError: mongoose is not defined\n    at ConnectToDatabase (file:///home/emmanuel/kitchen/work-project/questn/src/utils/database.js:17:5)\n    at file:///home/emmanuel/kitchen/work-project/questn/server.js:17:1\n    at ModuleJob.run (node:internal/modules/esm/module_job:218:25)\n    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)\n    at async loadESM (node:internal/process/esm_loader:28:7)\n    at async handleMainPromise (node:internal/modules/run_main:113:12)"}
{"level":"error","message":"ValidationError: User validation failed: fullname: Path `fullname` is required.\n    at Document.invalidate (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/document.js:3306:32)\n    at /home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/document.js:3067:17\n    at /home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/schemaType.js:1388:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"}
{"level":"error","message":"ReferenceError: verificationToken is not defined\n    at signup (file:///home/emmanuel/kitchen/work-project/questn/src/modules/auth/controller.js:19:34)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"ReferenceError: verificationToken is not defined\n    at signup (file:///home/emmanuel/kitchen/work-project/questn/src/modules/auth/controller.js:19:34)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"ReferenceError: generateRefreshToken is not defined\n    at login (file:///home/emmanuel/kitchen/work-project/questn/src/modules/auth/controller.js:41:26)"}
{"level":"error","message":"ValidationError: Token validation failed: user_id: Path `user_id` is required.\n    at Document.invalidate (/home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/document.js:3306:32)\n    at /home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/document.js:3067:17\n    at /home/emmanuel/kitchen/work-project/questn/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/schemaType.js:1388:9\n    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)"}
{"level":"error","message":"ReferenceError: setCookie is not defined\n    at login (file:///home/emmanuel/kitchen/work-project/questn/src/modules/auth/controller.js:43:5)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"ReferenceError: setCookie is not defined\n    at login (file:///home/emmanuel/kitchen/work-project/questn/src/modules/auth/controller.js:43:5)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"ReferenceError: setCookie is not defined\n    at login (file:///home/emmanuel/kitchen/work-project/questn/src/modules/auth/controller.js:43:5)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"}
{"level":"error","message":"Error connecting to database The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.","stack":"MongooseError: The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.\n    at NativeConnection.createClient (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:219:11)\n    at NativeConnection.openUri (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/connection.js:823:34)\n    at Mongoose.connect (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/mongoose.js:429:15)\n    at ConnectToDatabase (file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/src/utils/database.js:18:20)\n    at file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/server.js:17:1\n    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)\n    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)\n    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)"}
{"level":"error","message":"Error connecting to database The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.","stack":"MongooseError: The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.\n    at NativeConnection.createClient (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:219:11)\n    at NativeConnection.openUri (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/connection.js:823:34)\n    at Mongoose.connect (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/mongoose.js:429:15)\n    at ConnectToDatabase (file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/src/utils/database.js:18:20)\n    at file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/server.js:17:1\n    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)\n    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)\n    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)"}
{"level":"error","message":"Error connecting to database The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.","stack":"MongooseError: The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.\n    at NativeConnection.createClient (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:219:11)\n    at NativeConnection.openUri (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/connection.js:823:34)\n    at Mongoose.connect (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/mongoose.js:429:15)\n    at ConnectToDatabase (file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/src/utils/database.js:18:20)\n    at file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/server.js:17:1\n    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)\n    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)\n    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)"}
{"level":"error","message":"Error connecting to database The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.","stack":"MongooseError: The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.\n    at NativeConnection.createClient (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:219:11)\n    at NativeConnection.openUri (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/connection.js:823:34)\n    at Mongoose.connect (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/mongoose.js:429:15)\n    at ConnectToDatabase (file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/src/utils/database.js:18:20)\n    at file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/server.js:17:1\n    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)\n    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)\n    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)"}
{"level":"error","message":"Error connecting to database The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.","stack":"MongooseError: The `uri` parameter to `openUri()` must be a string, got \"undefined\". Make sure the first parameter to `mongoose.connect()` or `mongoose.createConnection()` is a string.\n    at NativeConnection.createClient (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:219:11)\n    at NativeConnection.openUri (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/connection.js:823:34)\n    at Mongoose.connect (/Users/brain/Desktop/kitchen/emmanuel/questn-api-service/node_modules/.pnpm/mongoose@8.4.4/node_modules/mongoose/lib/mongoose.js:429:15)\n    at ConnectToDatabase (file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/src/utils/database.js:18:20)\n    at file:///Users/brain/Desktop/kitchen/emmanuel/questn-api-service/server.js:17:1\n    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)\n    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)\n    at async asyncRunEntryPointWithESMLoader (node:internal/modules/run_main:123:5)"}
